tests/Js/EventsTest.php

```diff
@@ -56,6 +56,15 @@ public function testFocus()
 
         $focusBlurDetector->focus();
         $this->assertEquals('focused', $focusBlurDetector->getValue());
+
+        $focusableAnchor = $this->getAssertSession()->elementExists('css', '.elements a#focusable');
+        $this->assertEquals('no action detected', $focusableAnchor->getText());
```

Comment:
This should be added as a separate tests. We want granular tests in that testsuite to make it easier to skip them when needed for unsupported things in each driver.